src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+function centerSpider(first, second) {
```

Comment:
The function 'centerSpider' has unused arguments 'first' and 'second'. You should remove them and use the globally declared variables 'wall' and 'spider' directly inside the function.